apps/native-component-list/package.json

```diff
@@ -89,7 +92,7 @@
     "react-native-gesture-handler": "~1.6.0",
     "react-native-is-iphonex": "^1.0.1",
     "react-native-maps": "0.27.1",
-    "react-native-paper": "github:brentvatne/react-native-paper#@brent/fix-bottom-navigation-rn-57",
```

Comment:
What does this do? Will it conflict with V5?

Reply:
it looks like this was just a fix for https://github.com/react-navigation/react-navigation/issues/7466 and we should be able to update react-native-paper to latest